paywall/src/unlock.js/setupPostOffices.ts

```diff
@@ -15,26 +15,38 @@ interface process {
 declare const process: process
 
 export function normalizeConfig(unlockConfig: any) {
+  const unlockUserAccounts =
+    unlockConfig.unlockUserAccounts === 'true' ||
+    unlockConfig.unlockUserAccounts === true
+
   if (
     !unlockConfig ||
     !unlockConfig.locks ||
     typeof unlockConfig.locks !== 'object'
   )
-    return unlockConfig
+    return {
```

Comment:
I'm surprised typescript allowed this through! There is a chance that `unlockConfig` could be not an object, like `NaN` or `null` here, so this code will crash in that case. If the config is not valid, we shouldn't touch it, as it will fail validation later anyways.